core/src/main/resources/org/apache/spark/ui/static/sorttable.js

```diff
@@ -169,7 +169,7 @@ sorttable = {
     for (var i=0; i<table.tBodies[0].rows.length; i++) {
       text = sorttable.getInnerText(table.tBodies[0].rows[i].cells[column]);
       if (text != '') {
-        if (text.match(/^-?[�$�]?[\d,.]+%?$/)) {
+        if (text.match(/^-?[�$�]?[\d,.]+%?$/)) {
```

Comment:
My bad Sean, you are right: it's not intended (resulting upgrade to latest sorttable version). Let me fix that.